libvdrtools/indy-wallet/src/lib.rs

```diff
@@ -704,6 +704,61 @@ impl WalletService {
         Ok(())
     }
 
+    pub async fn migrate_records<E>(
```

Comment:
Most of the internal vdrtools wallet API is not public (I assume by design, so that weird things cannot be done from outside). So to have this someplace else means exposing a lot of the wallet API, which I'm not necessarily against but don't find as a good idea either.

There's no reason this wouldn't work for credx -> anoncreds-rs. This is the wallet API, not the anoncreds API.